composer.json

```
@@ -25,8 +25,8 @@
         "sensiolabs/security-checker": " ^5.0",
         "sylius/sylius": "dev-master",
         "symfony/flex": "^1.2",
-        "sylius/invoicing-plugin": "^0.8.3",
-        "sylius/refund-plugin": "^0.6.0",
+        "sylius/invoicing-plugin": "^0.10",
+        "sylius/refund-plugin": "^1.0.0-RC",
```

We should use this notation:
```suggestion
        "sylius/refund-plugin": "^1.0.0@rc",
```
to support upgrades for this stability. Ref. https://getcomposer.org/doc/articles/versions.md#stability-constraints

What is more, I highly recommend checking versions on https://semver.mwl.be/#!?package=sylius%2Frefund-plugin&version=%5E1.0&minimum-stability=RC